### PR TITLE
cleanup chart propnames & rename xCanvas -> xRenderable

### PR DESCRIPTION
**Pull request by allanhortle**

### CODE REVIEW

File: src/component/Chart.jsx [hunk 1]
```diff
@@ -1,57 +1,76 @@
...
+import Svg from './Svg';
 import * as d3Scale from 'd3-scale';
 
 
 /**
  * @component
  *
  * Chart is an organizer for D3's scales. Using the dimensions array it constructs a number of
- * scales, then it checks the children props and applies the scales accordingly.
+ * scales. It checks it's children's props and applies to them these scales configurations it found.
```
Comment (dancoates) [inline src/component/Chart.jsx]: no apostrophe for possessive its.

Comment (dancoates) [reply src/component/Chart.jsx]: Also, "to them these scales configurations it found"

File: src/component/Chart.jsx [hunk 2]
```diff
@@ -1,57 +1,76 @@
...
  * e.g. `<Chart xScaleType="scalePoint" yScaleType="scaleLinear" />`
  *
- * @prop {function} [<scaleName>Scale]
+ * @prop {function} [<dimensionName>Scale]
  * Called with scale and props after the default scale has been constructed
  * e.g. `<Chart xScaleType={scale => scale.padding(.1)} />`
  *
+ * @prop {function} [<dimensionName>ScaleGroup]
+ * Sometimes a dimension might have multiple scales. Scale groups isolate columns into different scales.
+ * Useage examples might be a correlation graph with independant y axis.
```
Comment (dancoates) [inline src/component/Chart.jsx]: Should probably be "independant y axes"

**Review by dancoates — CHANGES_REQUESTED** [review]
A few super pedantic things

**Review by dxinteractive — APPROVED** [review]

File: src/component/Chart.jsx (hunk 2 above)
Comment (dxinteractive) [reply src/component/Chart.jsx]: And "independent" not "independant" haha

Comment (dxinteractive) [reply src/component/Chart.jsx]: And "Usage" not "Useage"

**Review by dancoates — APPROVED** [review]

**Pull request merged**